packages/graphql/test/activityPermissions.test.ts

```
@@ -30,7 +37,7 @@ describe('Unit tests covering the creation of derived permissions for activities
   })
 
   beforeEach(async () => {
-    await testInitialization(prisma, emitter)
+    const {} = await testInitialization(prisma, hatchet, emitter)
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)